src/backend/looped/weapons/flying_axe.cpp

```diff
@@ -0,0 +1,254 @@
+#include "backend/looped_command.hpp"
+#include "natives.hpp"
+#include "util/timer.hpp"
+#include "script.hpp"
+#include "util/entity.hpp"
+#include "gta/enums.hpp"
+namespace big
+{
+	static timer spawning_axe_delay(std::chrono::milliseconds(200));
```

Comment:
Why are you force-casting the timer's delay parameter to nanoseconds, only to duration_cast it in the constructor to ::duration?

Reply:
Fixed